.circleci/scripts/deps-install.sh

```diff
@@ -5,7 +5,7 @@ set -x
 # Exit immediately if a command exits with a non-zero status.
 set -e
 
-yarn install --frozen-lockfile --har
+yarn install --frozen-lockfile
 
 # Move HAR file into directory with consistent name so that we can cache it
 mkdir -p build-artifacts/yarn-install-har
```

Comment:
if you're removing the har file generation, remove all references to the file

Reply:
the intention of the har inclusion is to improve the trail of how code entered the build, but this particular file may not be high value

Reply:
Good shout out